unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/Tests/RPCSocialApiBridgeShould.cs

```diff
@@ -0,0 +1,124 @@
+                                             Message = "",
+                                             CreatedAt = 1678912696639
+                                         }
+                                     }
+                                 }
+                             },
+                             Outgoing = new Requests()
+                             {
+                                 Items =
+                                 {
+                                     new List<RequestResponse>()
+                                     {
+                                         new RequestResponse()
+                                         {
+                                             User = new User() { Address = "addr3" },
+                                             Message = "",
+                                             CreatedAt = 1678912696639
+                                         },
+                                         new RequestResponse()
+                                         {
+                                             User = new User() { Address = "addr4" },
+                                             Message = "",
+                                             CreatedAt = 1678912696639
+                                         }
+                                     }
+                                 }
+                             }
+                         }));
+
+            UserProfile.GetOwnUserProfile().UpdateData(new UserProfileModel() { userId = OWN_ID });
+
+            GameObject go = new GameObject();
+            var component = go.AddComponent<MatrixInitializationBridge>();
+            var userProfileBridge = Substitute.For<IUserProfileBridge>();
+            rpcSocialApiBridge = new RPCSocialApiBridge(component, userProfileBridge);
+        }
+
+        // [UnityTest]
+        // public IEnumerator Initialize()
+        // {
+        //     return UniTask.ToCoroutine(async () =>
+        //     {
+        //         var cancellationToken = new CancellationToken();
+        //         var friends = new Dictionary<string, UserStatus>();
+        //         var friendRequests = new Dictionary<string, FriendRequest>();
+        //
+        //         rpcSocialApiBridge.OnFriendAdded += friend => { friends.Add(friend.userId, friend); };
+        //         rpcSocialApiBridge.OnFriendRequestAdded += request => { friendRequests.Add(request.FriendRequestId, request); };
+        //
+        //         await rpcSocialApiBridge.InitializeFriendshipsInformation(cancellationToken);
+        //
+        //         Assert.AreEqual(4, friends.Count);
+        //         Assert.AreEqual(2, friendRequests.Count(friendRequest => friendRequest.Value.From == OWN_ID));
+        //         Assert.AreEqual(2, friendRequests.Count(friendRequest => friendRequest.Value.To == OWN_ID));
+        //     });
+        // }
+    }
```

Comment:
We are missing an actual test suite, remove this commented code and implement a good coverage of the service.